### PR TITLE
Feature/add new1

### PR DESCRIPTION
**Pull request by ntsanq**
@ntsanqnfq check this plzz

### CODE REVIEW

**Comment by ntsanqnfq** [comment]
nooo

File: new2.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+new 2
+new 2
+new 2
+new 2
+new 2
+new 2
+new 2
+new 2
+new 2
+new 2
```
Comment (ntsanqnfq) [inline new2.txt:10]: change to new3

File: new2.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (ntsanqnfq) [inline new2.txt:2]: sdfsádas

**Comment by ntsanq** [comment]
ok

File: new2.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+new 2
+new 2
+new 3
+new 2
+new 2
+new 2
+new 2
+new 2
```
Comment (ntsanqnfq) [inline new2.txt:16]: add an image here

**Comment by ntsanqnfq** [comment]
ok

**Pull request merged**